pep-0307.txt

```diff
@@ -5,767 +5,817 @@ Last-Modified: $Date$
 Author: Guido van Rossum, Tim Peters
 Status: Final
 Type: Standards Track
-Content-Type: text/plain
+Content-Type: text/x-rst
 Created: 31-Jan-2003
 Post-History: 7-Feb-2003
 
-
 Introduction
+============
 
-    Pickling new-style objects in Python 2.2 is done somewhat clumsily
-    and causes pickle size to bloat compared to classic class
-    instances.  This PEP documents a new pickle protocol in Python 2.3
-    that takes care of this and many other pickle issues.
+Pickling new-style objects in Python 2.2 is done somewhat clumsily
+and causes pickle size to bloat compared to classic class
+instances.  This PEP documents a new pickle protocol in Python 2.3
+that takes care of this and many other pickle issues.
 
-    There are two sides to specifying a new pickle protocol: the byte
-    stream constituting pickled data must be specified, and the
-    interface between objects and the pickling and unpickling engines
-    must be specified.  This PEP focuses on API issues, although it
-    may occasionally touch on byte stream format details to motivate a
-    choice.  The pickle byte stream format is documented formally by
-    the standard library module pickletools.py (already checked into
-    CVS for Python 2.3).
+There are two sides to specifying a new pickle protocol: the byte
+stream constituting pickled data must be specified, and the
+interface between objects and the pickling and unpickling engines
+must be specified.  This PEP focuses on API issues, although it
+may occasionally touch on byte stream format details to motivate a
+choice.  The pickle byte stream format is documented formally by
+the standard library module ``pickletools.py`` (already checked into
+CVS for Python 2.3).
 
-    This PEP attempts to fully document the interface between pickled
-    objects and the pickling process, highlighting additions by
-    specifying "new in this PEP".  (The interface to invoke pickling
-    or unpickling is not covered fully, except for the changes to the
-    API for specifying the pickling protocol to picklers.)
+This PEP attempts to fully document the interface between pickled
+objects and the pickling process, highlighting additions by
+specifying "new in this PEP".  (The interface to invoke pickling
+or unpickling is not covered fully, except for the changes to the
+API for specifying the pickling protocol to picklers.)
 
 
 Motivation
+==========
 
-    Pickling new-style objects causes serious pickle bloat.  For
-    example,
+Pickling new-style objects causes serious pickle bloat.  For
+example,
 
-        class C(object): # Omit "(object)" for classic class
-            pass
-        x = C()
-        x.foo = 42
-        print len(pickle.dumps(x, 1))
+.. code-block:: python
```

Comment:
I think @ilevkivskyi's PR at pythondotorg repo is meant to address this https://github.com/python/pythondotorg/pull/1063

I haven't had chance to review / test it locally ..